sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -448,4 +456,120 @@ export class ApplicationStudentsController extends BaseController {
       applicationNumber: application.applicationNumber,
     };
   }
+
+  /**
+   * Get in progress details of an application by application id.
+   * @param applicationId application id.
+   * @returns application details.
+   */
+  @Get(":id/in-progress")
```

Comment:
Minor one. we can use  applicationId here. 

`@Get(":applicationId/in-progress")`